src/components/Payroll/PayrollOverview/PayrollOverviewPresentation.tsx

```diff
@@ -8,55 +10,60 @@ interface PayrollOverviewProps {
 
 export const PayrollOverviewPresentation = ({ onEdit, onSubmit }: PayrollOverviewProps) => {
   const { Alert, Button, Heading, Text } = useComponentContext()
+  useI18n('Payroll.PayrollOverview')
+  const { t } = useTranslation('Payroll.PayrollOverview')
 
   return (
     <Flex flexDirection="column" alignItems="stretch">
       <Flex justifyContent="space-between">
-        <Heading as="h1">Review payroll for Jul 5 - Jul 18, 2025</Heading>
+        <Heading as="h1">{t('pageTitle', { startDate: 'Jul 5', endDate: 'Jul 18, 2025' })}</Heading>
```

Comment:
You may want to communicate with the team before adding this one? i think dmitriy may be working in this file?